apps/climatemappedafrica/src/lib/data/blockify/hero.js

```diff
@@ -0,0 +1,32 @@
+import {
+  fetchProfile,
+  fetchProfileGeography,
+} from "@/climatemappedafrica/lib/hurumap";
+
+export default async function hero(block) {
+  const { geometries } = await fetchProfileGeography(
+    block.location?.name?.toLowerCase() ?? "af",
+  );
+  const { level } = geometries.boundary?.properties ?? {};
+  const childLevelMaps = {
+    continent: "country",
+    country: "region",
+  };
+  const childLevel = childLevelMaps[level];
+  const { locations } = await fetchProfile();
+  const featuredLocations = locations.filter(
+    (location) => location.level === childLevel,
+  );
+
+  const { children } = geometries;
+  const boundary =
+    children[level === "country" ? "County" : childLevel] ?? null;
```

Comment:
Why this mismatch of `country` and `County`?

Reply:
@kilemensi because this is how children data looks like when the selected level is Country
```
{ County: { type: 'FeatureCollection', features: [] } }
```
at least for Kenya

Reply:
Other countries don't have counties @koechkevin